src/Microsoft.PowerShell.Commands.Diagnostics/GetEventCommand.cs

```diff
@@ -1454,18 +1455,19 @@ private string HandleStartTimeHashValue(Object value, Hashtable hash)
                 endTime = endTime.ToUniversalTime();
                 string endTimeFormatted = endTime.ToString("s", CultureInfo.InvariantCulture) + "." + endTime.Millisecond.ToString("d3", CultureInfo.InvariantCulture) + "Z";
 
-                ret += string.Format(CultureInfo.InvariantCulture,
-                                             "(System/TimeCreated[@SystemTime&gt;='{0}' and @SystemTime&lt;='{1}'])",
-                                             startTimeFormatted, endTimeFormatted);
+                ret.AppendFormat(CultureInfo.InvariantCulture,
+                                 SystemTimeTemplate2,
+                                 startTimeFormatted,
+                                 endTimeFormatted);
             }
             else
             {
-                ret += string.Format(CultureInfo.InvariantCulture,
-                                             "(System/TimeCreated[@SystemTime&gt;='{0}'])",
-                                             startTimeFormatted);
+                ret.AppendFormat(CultureInfo.InvariantCulture,
+                                 SystemTimeTemplate,
```

Comment:
The template you use here is wrong. The original template here is &**gt;**, but `SystemTimeTemplate` is &**lt;**